agents/habitat_baselines_agent.py

```diff
@@ -308,7 +297,9 @@ def main():
         "--evaluation", type=str, required=True, choices=["local", "remote"]
     )
     parser.add_argument("--model-path", default="", type=str)
-    parser.add_argument("--task", required=True, type=str, choices=["objectnav", "imagenav"])
+    parser.add_argument(
+        "--task", required=True, type=str, choices=["objectnav", "instance_imagenav"]
```

Comment:
instance_imagenav -> imagenav

Reply:
Sorry I think this happened due to some automatic merging of our branches.